src/components/WriteEditModal/KeyboardInfoBox.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import classNames from 'classnames/bind';
+import { CustomKeyboardTypes } from '@/app/types/CustomKeyboardTypes';
+import styles from './KeyboardInfoBox.module.scss';
+
+interface KeyboardInfoBoxProps {
+  keyboardInfo: CustomKeyboardTypes;
+  isReview?: boolean;
+}
+
+export default function KeyboardInfoBox({ keyboardInfo, isReview }: KeyboardInfoBoxProps) {
+  const cn = classNames.bind(styles);
+  const {
+    layout,
+    appearance_texture: appearanceTexture,
+    appearance_color: appearanceColor,
+    switch: keyboardSwitch,
+    has_point_key: hasPointKey,
+    keycap_color: keycapColor,
+  } = keyboardInfo;
+  const optionString = `${layout} 
+                        / ${appearanceTexture} 
+                        / ${appearanceColor} 
+                        / ${keyboardSwitch} 
+                        / 포인트 키캡 ${hasPointKey ? 'o' : 'x'}
+                      `;
+  let keycapColors = '';
+
+  Object.entries(keycapColor).forEach(([key, val]) => {
+    keycapColors += `${key}: ${val}`;
+  });
```

Comment:
let사용을 피하는 게 좋습니다!
저라면 JSON.stringify와 replaceAll을 사용할 것 같아요
```suggestion
  const keycapData = JSON.stringify(keycapColor).replaceAll('"', '');
  const keycapColors = keycapData.slice(1, keycapData.length - 1);

```

Reply:
이런 방법이 있었군요,,, 대박,, 네네 참고하겠습니다!!! 🙆‍♀️

Reply:
오호,, 도원님 리뷰 읽다보니 저라면 이런 방법을 쓸 거 같아용!!
```suggestion
const keycapColors = Object.entries(keycapColor)
  .map(([key, val]) => `${key}: ${val}`)
  .join(', ');
```

Reply:
오옷!! 이것도 간단하게 좋은 방법인 것 같아요오!! 참고하겠습니다!! 🙆‍♀️